runtime/LibcWrappers.cpp

```diff
@@ -246,6 +246,26 @@ size_t SYM(fread)(void *ptr, size_t size, size_t nmemb, FILE *stream) {
   return result;
 }
 
+char *SYM(fgets)(char *str, int n, FILE *stream) {
+  tryAlternative(str, _sym_get_parameter_expression(0), SYM(fgets));
+  tryAlternative(n, _sym_get_parameter_expression(1), SYM(fgets));
+
+  auto result = fgets(str, n, stream);
+  _sym_set_return_expression(nullptr);
```

Comment:
`fgets` returns `str` on success, so I suppose we should do `_sym_set_return_expression(_sym_get_parameter_expression(0))` in that case to make sure that the analysis understands the return value if it's used later on.

Reply:
I applied the suggested changes :)